components/ResilienceResource/ColourBox.js

```diff
@@ -21,7 +22,23 @@ function ColourBox({
   width,
   height,
   margin,
+  format,
 }) {
+  let content
+  if (format === 'points') {
+    content = (
+      <ul>
+        {points.map((point) => {
```

Comment:
next time you can just use round brackets instead of curly ones and you won't have to write the return keyword